libs/dkg/src/rbc.cpp

```diff
@@ -73,11 +73,21 @@ RBC::RBC(Endpoint &endpoint, MuddleAddress address, CabinetMembers const &cabine
     RBCSerializer serialiser(payload);
 
     // Deserialize the RBCEnvelope
-    RBCEnvelope env;
-    serialiser >> env;
+    RBCMessage msg;
+    try
+    {
+      serialiser >> msg;
+    }
+    catch (...)
+    {
+      std::cout << "PANIC! PANIC! PANIC! PANIC! PANIC! PANIC! PANIC! PANIC! PANIC! PANIC! "
```

Comment:
use logger_error here. also best to catch specific exceptions right

Reply:
Hmm... that was supposed to be gone. Also, error message is not all that good!

Reply:
```suggestion
```